src/room.html

```diff
@@ -149,6 +156,9 @@
             </a-gltf-entity>
         </a-entity>
 
+        <!-- Lights -->
+        <a-entity light="type: directional; color: #F9FFCE; intensity: 0.6" position="0.002 5.231 -15.3"></a-entity>
```

Comment:
We probably want a way to only include this in "high" quality mode

Reply:
Hmm. How do we want to do that? More templates?